modules/apps/notification/notification-web/src/main/java/com/liferay/notification/web/internal/portlet/action/NotificationTemplateGeneralTermsMVCResourceCommand.java

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.notification.web.internal.portlet.action;
+
+import com.liferay.notification.constants.NotificationPortletKeys;
+import com.liferay.object.definition.notification.term.util.ObjectDefinitionNotificationTermUtil;
+import com.liferay.portal.kernel.portlet.bridges.mvc.MVCResourceCommand;
+import com.liferay.portal.kernel.util.HashMapBuilder;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.Set;
+
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author Paulo Albuquerque
+ */
+@Component(
+	property = {
+		"javax.portlet.name=" + NotificationPortletKeys.NOTIFICATION_TEMPLATES,
+		"mvc.command.name=/notification_templates/notification_template_general_terms"
+	},
+	service = MVCResourceCommand.class
+)
+public class NotificationTemplateGeneralTermsMVCResourceCommand
```

Comment:
@p-albuquerque 

rename to `GeneralNotificationTemplateTermsMVCResourceCommand`